input/images-source/DeniedSubscriptionSequence.plantuml

```diff
@@ -1,24 +1,27 @@
 @startuml
 
-title Websocket subscription denial
+title WebSocket subscription denial flow diagram
 
 participant "App\n(Subscriber)" as Subscriber
 participant "FHIRcast\nHub" as Hub
 
 Subscriber -> Hub++: POST Suscribe
 Subscriber <-- Hub--: 202 Accepted [<wss URL>]
 Subscriber -> Hub: connect to <wss URL>>
-Subscriber <-- Hub: WS: conformation
+alt with confirmation

```

Comment:
Does a hub send a confirmation in the denial workflow?  The 'Subscription Confirmation' section says "If a subscribe or unsubscribe request is not denied...", so I do not think it should.

I believe this entire section can be replaced with:
Hub --> Subscriber: WSS: denial
(note that the `++` should also be removed, since the connection is denied and not active)

Reply:
Gino, will you look at out modification?

Optimally, this diagram would represent both synchronous and asynchonous denial.